src/workerd/server/BUILD.bazel

```diff
@@ -37,7 +37,7 @@ wd_cc_library(
     name = "malloc",
     visibility = ["//visibility:public"],
     deps = select({
-        ":really_use_tcmalloc": ["@com_google_tcmalloc//tcmalloc"],
+        ":really_use_tcmalloc": ["@tcmalloc//tcmalloc"],
```

Comment:
The internal build is failing since it doesn't recognize the `@ tcmalloc` dependency. I guess using `repo_name = "com_google_tcmalloc"` would be a good workaround for now